### PR TITLE
create Hosted Snippet Custom Bundle File documentation

### PR DESCRIPTION
**Pull request by williankargo**
*Issue #, if available:*
- https://github.com/amazon-connect/amazon-connect-chat-ui-examples/pull/141
- https://github.com/amazon-connect/amazon-connect-chat-interface/pull/143
- https://github.com/amazon-connect/amazon-connect-chat-interface/pull/161
- https://github.com/amazon-connect/amazon-connect-chat-interface/pull/184

*Description of changes:*
I gonna change the document on the link: https://github.com/amazon-connect/amazon-connect-chat-ui-examples/tree/master/cloudformationTemplates/startChatContactAPI
by removing #prebuild-chat-widget from the original url, users can see my new description which is essential to read at the beginning of the doc.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by spencerlepine — CHANGES_REQUESTED** [review]
.. outdated comment

**Review by spencerlepine — CHANGES_REQUESTED** [review]

File: .github/docs/built-in-script-setup-setps.md
[the remark this answers is not in the document]
Comment (williankargo) [reply .github/docs/built-in-script-setup-setps.md:1]: yes

File: .github/docs/built-in-script-setup-setps.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Use the customized version of amazon-connect-chat-interface.js in a quick way
```
[the remark this answers is not in the document]
Comment (williankargo) [reply .github/docs/built-in-script-setup-setps.md:1]: yes. and I think we should only put one code section in the document so customers won't feel confused.

File: .github/docs/screenshoots/built-in-script-setup-steps-change-color.jpg
[the remark this answers is not in the document]
Comment (williankargo) [reply .github/docs/screenshoots/built-in-script-setup-steps-change-color.jpg:1]: ok

**Review by spencerlepine — APPROVED** [review]
🚢

**Comment by spencerlepine** [comment]
Note for later: need to merge https://github.com/amazon-connect/amazon-connect-chat-interface/pull/161 afterwards

File: .github/docs/ HostedSnippetCustomBundleFileSetup.md.md
Comment (spencerlepine) [inline .github/docs/ HostedSnippetCustomBundleFileSetup.md.md:1]: Could we fix file name to be: `HostedSnippetCustomBundleFileSetup.md`?

**Review by spencerlepine — CHANGES_REQUESTED** [review]

File: .github/docs/ HostedSnippetCustomBundleFileSetup.md.md
Comment (williankargo) [reply .github/docs/ HostedSnippetCustomBundleFileSetup.md.md:1]: yes

**Review by spencerlepine — APPROVED** [review]

**Comment by spencerlepine** [comment]
-Need to remove `%20` before file name
-Need to update # comment with // for javascript

File: .github/docs/ HostedSnippetCustomBundleFileSetup.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+## Setup
+1. If you don't already have a Chat Widget instance in your AWS account, please set one up by following this [AWS Guide](https://docs.aws.amazon.com/connect/latest/adminguide/tutorial1-set-up-your-instance.html).
+
+2. On the Chat Widget Panel, you can see your Chat widget script.
+![Alt text](screenshoots/built-in-script-setup-steps-lmaw.jpg)
+
+3. You can customize and upload your `amazon-connect-interface.js` file to your own S3 bucket or any storage server. Follow [customize interface.js steps](https://github.com/amazon-connect/amazon-connect-chat-interface/blob/spenlep/improve-readme-documentation/DOCUMENTATION.md#features) to customize `amazon-connect-interface.js`.
+
+    - 3-1. Build your own amazon-connect-chat-interface.js and upload it in your S3 bucket.
+    ![Alt text](screenshoots/built-in-script-setup-steps-s3.jpg)
```
Comment (spencerlepine) [inline .github/docs/ HostedSnippetCustomBundleFileSetup.md:31]: need to rename to screenshots not screenshoots

File: .github/docs/ HostedSnippetCustomBundleFileSetup.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    ![Alt text](screenshoots/built-in-script-setup-steps-s3-public.jpg)
+
+4. Now, with just one line of script added to your Chat Widget script, you can use `amazon-conncet-chat-interface.js` in your chat widget.
+    - 4-1. Copy the link to customized `amazon-connect-chat-interface.js`.
+    ![Alt text](screenshoots/built-in-script-setup-steps-s3-url.jpg)
+
+    - 4-2. Use the link as a parameter in the following script. Include this script along with your Chat Widget script and paste it into your website file.
+
+        ```
+        amazon_connect('customerChatInterfaceUrl', 'https://...'); # TODO: put in your link to amazon-connect-chat-interface.js
```
Comment (spencerlepine) [inline .github/docs/ HostedSnippetCustomBundleFileSetup.md:42]: need to fix comment to use // not #

File: .github/docs/ HostedSnippetCustomBundleFileSetup.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+Host your own `amazon-connect-chat-interface.js` bundle file and provide the link in the widget snippet configuration. Retrieve this snippet code from the Connect Admin Console: [Documentation - "Chat widget script"](https://docs.aws.amazon.com/connect/latest/adminguide/add-chat-to-website.html#chat-widget-script)
+
+```diff
+<script type="text/javascript">
+  (function(w, d, x, id){s=d.createElement('script');s.src='https://d2s9x5slbvr0vu.cloudfront.net/amazon-connect-chat-interface-client.js';s.async=1;s.id=id;d.getElementsByTagName('head')[0].appendChild(s);w[x]=w[x]||function(){(w[x].ac=w[x].ac||[]).push(arguments)}})(window, document, 'amazon_connect', 'asdfasdfasdf');
+  
+  amazon_connect('styles', { openChat: { color: '#ffffff', backgroundColor: '#07b62a'}, closeChat: { color: '#ffffff', backgroundColor: '#07b62a'} });
+  amazon_connect('snippetId', 'asdfsadfasdf...');
+  amazon_connect('supportedMessagingContentTypes', [ 'text/plain', 'text/markdown' ]);
++ amazon_connect('customerChatInterfaceUrl', 'https://...'); # TODO: put in your link to amazon-connect-chat-interface.js
```
Comment (spencerlepine) [inline .github/docs/ HostedSnippetCustomBundleFileSetup.md:64]: need to fix comment to use // not #

File: .github/docs/ HostedSnippetCustomBundleFileSetup.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
+# Host Widget Snippet Integrated Custom UI (S3 Bucket/CDN)
```
Comment (spencerlepine) [inline .github/docs/ HostedSnippetCustomBundleFileSetup.md:1]: fix file name to be `HostedSnippet...` not ` HostedSnippet...`

**Review by spencerlepine — CHANGES_REQUESTED** [review]

File: .github/docs/screenshots/built-in-script-setup-steps-lmaw.jpg
Comment (spencerlepine) [inline .github/docs/screenshots/built-in-script-setup-steps-lmaw.jpg:1]: need to blur cloudfront

**Comment by spencerlepine** [comment]
Had to rename file, re-opened and co-authored in: https://github.com/amazon-connect/amazon-connect-chat-interface/pull/185

**Pull request closed without merging**